packages/patterns-react/src/patterns/SimpleOverview/SimpleOverview.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { DDS_SIMPLE_OVERVIEW } from '../../internal/FeatureFlags';
+import { ArrowRight20 } from '@carbon/icons-react';
+import { LinkWithIcon } from '@carbon/ibmdotcom-react';
+import classNames from 'classnames';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Simple Overview
+ *
+ * @param {object} props Props object
+ * @param {string} props.label Label for the Simple Overview Pattern
+ * @param {string} props.heading Heading for the Simple Overview Pattern
+ * @param {string} props.copy Copy for the Simple Overview Pattern
+ * @param {object} props.link  Link object {href, target, copy}
+ * @returns {*} Simple Overview Pattern JSX object
+ */
+const SimpleOverview = ({ label, heading, copy, link }) => {
+  /**
+   * Renders the link based on the props
+   *
+   * @param {object} link Link object
+   * @returns {*} link div JSX object
+   */
+  const _renderLink = link => {
+    if (link) {
+      return (
+        <div className={`${prefix}--simpleoverview__link-container`}>
+          <LinkWithIcon href={link.href} target={link.target}>
+            <span>{link.copy}</span> <ArrowRight20 />
+          </LinkWithIcon>
+        </div>
+      );
+    }
+    return false;
+  };
+
+  return featureFlag(
+    DDS_SIMPLE_OVERVIEW,
+    <section
+      data-autoid={`${stablePrefix}--simpleoverview`}
+      className={`${prefix}--simpleoverview`}>
+      <div className={`${prefix}--simpleoverview__container`}>
+        <div className={`${prefix}--simpleoverview__row`}>
+          <div
+            className={classNames(
+              `${prefix}--simpleoverview__col`,
+              `${prefix}--simpleoverview__label-container`
+            )}>
+            <h4 className={`${prefix}--simpleoverview__label`}>{label}</h4>
+          </div>
+          <div
+            className={classNames(
+              `${prefix}--simpleoverview__col`,
+              `${prefix}--simpleoverview__content-container`
+            )}>
+            <h5 className={`${prefix}--simpleoverview__heading`}>{heading}</h5>
+            <p className={`${prefix}--simpleoverview__content`}>{copy}</p>
+            {_renderLink(link)}
+          </div>
```

Comment:
In the page hierarchy I'm wondering if this is an h2/h3? Any thoughts on this @kennylam @annawen1 @photodow @emerson-pereira @andysherman2121 @mkothur @carthick @sannminwin

Reply:
This is a good topic, I think that as the H1 and H2  are commonly used in the Leadspace. Maybe a H3 and H4 would be more appropriate, as a second thought

Reply:
@wonilsuhibm @larahanlon2 @oliviaflory where do you envision this pattern to typically be on a page?

Reply:
@jeffchew @sannminwin yes. This pattern will typically be right below the lead space pattern. Lead space presents the super high level overview and entices the user, then this pattern give a slightly deeper and better over view of the content of the page.

Reply:
based on what wonil pointed out i'll agree with rapha, i think h3/h4 would make more sense

Reply:
I've changed it to H3 and H4 in my last commit.